org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/outline/ModelBuilder.scala

```diff
@@ -0,0 +1,435 @@
+package org.scalaide.ui.internal.editor.outline
+/**
```

Comment:
This comment should be above the `ModelBuilder` type, otherwise Scaladoc can't find it.